operations/finance.md

```diff
@@ -254,7 +254,7 @@ Returns all accounting categories of the enterprise associated with the connecto
 
 ## Get all accounting items
 
-Returns all accounting items of the enterprise that were consumed \(posted\) or will be consumed within the specified interval. If the `Currency` is specified, costs of the items are converted to that currency.
+Returns all accounting items of the enterprise that were consumed \(posted\) or will be consumed within the specified interval. If the `Currency` is specified, costs of the items are converted to that currency. Accounting items are split into [Order item](#order-item)s (consumed items such as nights or products) and [Payment item](#payment-item)s (such as cash, credit card payments or invoices).
```

Comment:
Where are other changes related to other endpoints where the new structure is used?

Reply:
following prs. Written in the description